src/QSymbolicsBase/basic_ops_homogeneous.jl

```diff
@@ -202,7 +202,7 @@ julia> commutator(A, A)
     op1
     op2
     function SCommutator(o1, o2) 
-        coeff, cleanterms = prefactorscalings([o1 o2], scalar=true)
+        coeff, cleanterms = prefactorscalings([o1 o2])
```

Comment:
same question about dropping the custom constructor and moving this to `commutator`